### PR TITLE
[AV-324] write python script for rasaero data processing

### PR DESCRIPTION
**Pull request by ktt3**
added `utils/RASAero_fetch` directory to fetch the data from RASAero. `consolidate.py` can be run on any system, but `script.py` can only run on Windows

### CODE REVIEW

File: utils/RASAero_fetch/src/script.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
+import os
+import pyautogui
+import time
+from pathlib import Path
+
+def main():
+
+    out_pth = os.getcwd()[:-3] + "output\\"
```
Comment (NPIPHI) [inline utils/RASAero_fetch/src/script.py:8]: why do you cut off the last 3 characters from the path, it makes the path wrong on my machine

**Review by NPIPHI — CHANGES_REQUESTED** [review]
Watching it work is great, I don't think that all the 00_000.txt intermediate files should be pushed to the repo.

**Comment by ktt3** [comment]
> Watching it work is great, I don't think that all the 00_000.txt intermediate files should be pushed to the repo.

I feel like these are necessary. Because RASAero can only run on windows, it restricts users (like myself) from accessing this information. Leaving it in there could still keep the data accessible to Linux/MacOS users and other sub teams that might need to use this information.

File: utils/RASAero_fetch/src/script.py (hunk 1 above)
Comment (ktt3) [reply utils/RASAero_fetch/src/script.py:8]: @rishivg Might know more about this

**Comment by NPIPHI** [comment]
Good point, they should be kept in then

**Review by NPIPHI — APPROVED** [review]

**Pull request merged**